### PR TITLE
Added all my work from the weekend

### PR DESCRIPTION
**Pull request by mcmillanator**
TODO:
- [ ] Fix tab spacing.
- [ ] Add fallback font
- [ ] Fix CSS structure so !important can be removed
- [ ] Automate the column list
- [ ] find out why I'm hiding the <title> tag and fix it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                        </h2>
+                    </div>
+                </div>
+            </div>
+            <div class="col-5 no_pad">
+                <div class="row">
+                    <div class="col-5 article3">
+                        <h2>
+                            HERE'S ANOHTER LONGER POST TITLE
+                        </h2>
```
Comment (rpearce) [inline index.html]: can you keep your indent down to 2 spaces?

Comment (rpearce) [inline index.html]: it would help me read... and be internet awesome

File: sass/_base.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
+html {
+  display: block;
+  box-sizing: border-box;
+  font-family: 'Oswald', sans-serif;
```
Comment (rpearce) [inline sass/_base.scss]: what about a fallback font?

File: sass/_base.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+*,
+*:before,
+*:after {
+  display: inherit;
```
Comment (rpearce) [inline sass/_base.scss]: how come you're telling html to display block and then inheriting all the things here?

File: sass/_base.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  box-sizing: inherit;
+}
+title{
+  display: none;
+}
```
Comment (rpearce) [inline sass/_base.scss]: I don't think this makes much sense. The `<title>` tag is never shown, so why hide it with CSS?

File: sass/_base.scss [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  @include trans-background('./img/wallhaven-227260.jpg', inherit, 25, 118, 187, 0.6);
+}
+.right{
+  text-align: right;
+}
+.flex{
+  display: flex !important;
+}
+.no_pad{
+  padding: 0 !important;
```
Comment (rpearce) [inline sass/_base.scss]: any time I see `!important`, that tells me that the CSS hasn't been structured correctly and this is a hack to make it work

File: sass/_base.scss [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+}
+[class*='article'] {
+  min-height: 322px;
+  position: relative;
+  >*{
```
Comment (rpearce) [inline sass/_base.scss]: this is a dangerous selector when used like this, and i recommend never doing this. be specific

File: sass/_grid.scss [hunk 7]
```diff
@@ -0,0 +1,124 @@
+.grid-container{
+    width: 100%;
```
Comment (rpearce) [inline sass/_grid.scss]: not necessary if this selector is selecting a `div`

File: sass/_grid.scss [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+}
+.col-8{
+    width: 80%;  
+}
+.col-9{
+    width: 90%;  
+}
+.col-10{
+    width: 100%;  
+}
```
Comment (rpearce) [inline sass/_grid.scss]: I bet you could automate this list of columns with an `@each` block...

File: index.html (hunk 1 above)
Comment (mcmillanator) [inline index.html]: @rpearce I have "tab_size": 2 set in sublime.  Is that what you are talking about?  If so I'll have to see what's going on.  If not please clarify.  Thanks you.  :)

Comment (rpearce) [inline index.html]: Make sure to have the setting indent using spaces turned on

Comment (mcmillanator) [inline index.html]: I had that set to.  I did some googling.  I found out that I needed to add "detect_indentation": false, I think you may have asked us to set that one too, before I had to reinstall my OS.  I believe what happened was the OS default editor was used to create this file initially.  That editor saved some tab settings that sublime detected and used.  Turning that detection off fixed the problem.  :)

**Pull request merged**